### PR TITLE
More fixes for Ember 1.10

### PR DESCRIPTION
**Pull request by wagenet**

### CODE REVIEW

**Comment by bcardarella** [comment]
@rwjblue can you review this one? This was the stuff you handled

**Comment by rwjblue** [comment]
Yup was working on it with him in slack.

**Comment by bcardarella** [comment]
Good enough for me

File: app/helpers/t.js [hunk 1]
```diff
@@ -12,44 +8,8 @@ export default function tHelper() {
   var view = options.data.view;
   var container = view.container;
   var t = container.lookup('utils:t');
-  var application = container.lookup('application:main');
```
Comment (rwjblue) [inline app/helpers/t.js]: This is what allows rerender ing on changes to the locale. It needs to stay.

File: app/helpers/t.js [hunk 2]
```diff
@@ -12,44 +8,8 @@ export default function tHelper() {
...
     return t(path, args);
   });
-  var childView = new SimpleBoundView(stream, options.escaped);
-
-  stream.subscribe(view._wrapAsScheduled(function(){
-    Ember.run.scheduleOnce('render', childView, 'rerender');
-  }));
-
-  // bind any arguments that are Streams
-  for (var i = 0, l = args.length; i < l; i++) {
```
Comment (rwjblue) [inline app/helpers/t.js]: This block needs to stay also.

Comment (rwjblue) [inline app/helpers/t.js]: Or we don't rerender on changes to input streams.

File: app/helpers/t.js [hunk 3]
```diff
@@ -12,44 +8,8 @@ export default function tHelper() {
...
-    var param = args[i];
-    if(param && param.isStream){
-      param.subscribe(stream.notify, stream);
-    };
-  }
-
-  application.localeStream.subscribe(stream.notify, stream);
```
Comment (rwjblue) [inline app/helpers/t.js]: This needs to stay so we rerender on locale changes.

**Comment by bcardarella** [comment]
@rwjblue oh crap, I thought you meant you already reviewed in slack

**Comment by rwjblue** [comment]
I'll pull this down later tonight and play with it a bit. Figure out if we have any new edge cases.

**Comment by rwjblue** [comment]
Lol. No worries! I wasn't clear. We'll get it all fixed up tonight. It is MUCH closer.

**Pull request merged**